### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rbarillot**
Update Readme.md

### CODE REVIEW

**Comment by rbarillot** [comment]
I forgot to update the table of content and some links are still not defined

File: README.md [hunk 1]
```diff
@@ -1,31 +1,152 @@
...
+# Description
+WheatFspm consists in a set of sub-models (named submodules in git) which share inputs/outputs through an MTG object:
 
-## Developers
+![alt text](https://raw.githubusercontent.com/openalea-incubator/WheatFspm/master/WheatFspm.png "WheatFSPM workflow")
+*Adapted from Gauthier et al. (2020)*
 
-This  package contains [submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) ([in French](https://git-scm.com/book/fr/v2/Utilitaires-Git-Sous-modules)).
+* *Farquhar-Wheat*: Farquhar-based model of photosynthesis, stomatal conductance, organ temperature and transpiration.
+* *Elong-Wheat*: regulation of leaf and internode elongation by C and N metabolites and coordination rules.
```
Comment (mngauthier) [inline README.md:11]: C and N metabolites, temperatures and coordination rules.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,152 @@
...
+* Open a command line interpreter in `WheatFspm\fspm-wheat\example\Vegetative_stages`
+* Run script *main.py*: `python main.py`
+
+## Scenarii_monoculms
+Work in progress for simulating wheat plasticity to contrasted environments
+
+# Credits
+## Authors
+* Romain BARILLOT - model designing, development and validation - [rbarillot](https://github.com/rbarillot)
+* Marion GAUTHIER (Marion.Gauthier@inrae.fr) - model designing, development and validation - [mngauthier](https://github.com/mngauthier)
```
Comment (mngauthier) [inline README.md:137]: delete e-mail address

File: README.md [hunk 3]
```diff
@@ -1,31 +1,153 @@
...
+- [Description](#description)
+- [Table of Contents](#table-of-contents)
+- [Installation](#installation)
+  * [Prerequisites](#prerequisites)
+  * [Installing](#installing)
+    + [Users](#users)
+    + [Developers](#developers)
+- [Usage](#usage)
+  * [NEMA](#nema)
+  * [Papier_FSPMA2016](#papier-fspma2016)
```
Comment (pradal) [inline README.md:31]: Papier -> Paper

File: README.md [hunk 4]
```diff
@@ -1,31 +1,153 @@
...
+## NEMA
+This example deals with the post-flowering stages of wheat developement under 3 nitrogen fertilisation regies (H0, H3 and H15). The main processes described are leaf senescence, C and N remobilisation, grain filling). During that stages, all vegetative organs have completed their growth. 
+This work led to the research articles [Barillot *et al.* (2016a)](https://doi.org/10.1093/aob/mcw143) and [Barillot *et al.* (2016b)](https://doi.org/10.1093/aob/mcw144).
+This example has been maintained in the current version ; results of above papers were generated using the tag *release-1.0* of *CN-Wheat*. 
+
+To run the example:
+* Open a command line interpreter in `WheatFspm\fspm-wheat\example\NEMA`
+* Run each script called *main.py* located in *NEMA_H0*, *NEMA_H3*, *NEMA_H15*: `python main.py`
+
+## Papier_FSPMA2016
```
Comment (pradal) [inline README.md:113]: Replace Papier by Paper

File: README.md [hunk 5]
```diff
@@ -1,31 +1,153 @@
...
+This example deals with the effects of leaf inclination, radiations regimes, plant density and sowing patterns on plant metabolism and grain filling during the post-flowering stages.   
+This work led to the research article [Barillot *et al.* (2019)](https://academic.oup.com/aob/article/123/4/727/5236618).
```
Comment (pradal) [inline README.md:115]: Make a link with the doi rather than the web address

File: README.md (hunk 3 above)
Comment (rbarillot) [reply README.md:31]: Unfortunately, the directory was actually named that way

**Pull request merged**